### ISSUE TITLE
Credential caching "just works" on MacOS (mojave 10.14.6)

### ISSUE THREAD

**Issue by heilcw** [jennybc/happy-git-with-r#148]
Just installed Git and the credential caching "just works" on my mac

**Comment by hadley** [comment]
Same for me.

**Comment by jtr13** [comment]
Same here (MacOS Catalina 10.15.5). I tested out the line you have in the Windows section and got this:

```
joycerobbins@MacBook-Pro ~ % git config --show-origin --get credential.helper
file:/usr/local/etc/gitconfig	osxkeychain
```

The first time I did `git push` from RStudio I got a popup **git-credential-osxkeychain wants to access key "github.com" in your keychain** requesting the login password and choice to Always Allow, Deny, or Allow.

**Comment by jennybc** [comment]
Fixed in aebe80ef4f40b783dbab8229d72ab51ccf3f811e
